2-Webpack/scripts/todo_list.js

```diff
@@ -0,0 +1,237 @@
+import { LocalStorageManager } from "./localstorage_manager";
+const { v4: uuidv4 } = require("uuid");
+
+(function () {
+  const TYPE_ITEM_INPUT = "item-input";
+  const TYPE_ITEM_LABEL = "item-label"; 
+  const TYPE_EDIT_BTN = "edit";
+  const TYPE_REMOVE_BTN = "remove";
+  const TYPE_CHECK_BTN = "check";
+  const TYPE_CONFIRM_BTN = "confirm";
+
+  const iconMap = { edit: "&#xe3c9", remove: "&#xe14c;", check: "&#xe876;" };
+  const newItemInput = document.getElementById("new-item-name");
+  const addBtn = document.getElementById("add-btn");
+
+  let localStorageManager = new LocalStorageManager();
+
+  function createButton(type, cb) {
+    const button = document.createElement("div");
+    const icon = iconMap[type] || "&#xe145;";
+
+    const iconSpan = document.createElement("span");
+    iconSpan.classList.add("material-icons");
+    iconSpan.classList.add("md-36");
+    iconSpan.innerHTML = icon;
+
+    button.classList.add("btn");
+    button.classList.add(type + "-btn");
+    button.dataset.type = type;
+    button.appendChild(iconSpan);
+    button.onclick = cb;
```

Comment:
consider using `addEventListener` instead of overriding events via `onclick`, `ondoubleclick` and so on. 
- it can accept multiple handlers
- this specific function returns the button element, thus the returned element `onclick`s' property can be overridden and it's behavior will break